bulk-move.php

```diff
@@ -188,6 +191,8 @@ private function setup_actions() {
         // Register hooks
         add_action( 'admin_menu', array( &$this, 'add_menu' ) );
         add_action( 'admin_init', array( &$this, 'request_handler' ) );
+        add_action( 'wp_ajax_load_custom_taxonomy_by_post_type', 'Bulk_Move_Posts::load_custom_taxonomy_by_post_type' );
```

Comment:
Any specific reason for making these as static methods?

Reply:
All the methods in the `Bulk_Move_Posts` is static. To make it easier when testing, I stayed consistent with other methods.